post.js

```diff
@@ -98,7 +97,7 @@ class DistributedPost extends HTMLElement {
     try {
       const content = await db.getNote(postUrl)
       if (content && content.content) {
-        content.content = insertImagesAndVideos(content.content) // Resolve URLs before rendering
+        content.content = await insertImagesAndVideos(content.content) // Resolve URLs before rendering
```

Comment:
We should remove the async and just convert the img/video tags in a regular function since the load happens in the component.